code/modules/fallout/reagents/medicines.dm

```diff
@@ -136,10 +172,15 @@ datum/reagent/medicine/bitter_drink/on_mob_life(mob/living/M)
 	if(M.getBruteLoss() == 0 && M.getFireLoss() == 0 && M.getOxyLoss() == 0)
 		metabolization_rate = 1000 * REAGENTS_METABOLISM //instant metabolise if it won't help you, prevents prehealing before combat
 	if(!M.reagents.has_reagent(/datum/reagent/medicine/stimpak) && !M.reagents.has_reagent(/datum/reagent/medicine/healing_powder)) // We don't want these healing items to stack, so we only apply the healing if these chems aren't found. We only check for the less powerful chems, so the least powerful one always heals.
-		M.adjustFireLoss(-4*REAGENTS_EFFECT_MULTIPLIER)
-		M.adjustBruteLoss(-4*REAGENTS_EFFECT_MULTIPLIER)
-		M.adjustOxyLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
-		M.hallucination = max(M.hallucination, 5)
+		if(HAS_TRAIT(M, TRAIT_TECHNOPHOBE))
```

Comment:
See above for healing powder. Also, might even want to make it a subtype of healing powder.

Reply:
Changed poultice to be a subtype of healing powder.

```
/datum/reagent/medicine/healing_powder/poultice
	heal_factor = -2
	heal_factor_perk = 4
```